components/server/src/config/config-inferrer.ts

```diff
@@ -36,6 +37,11 @@ export class ConfigInferrer {
         return ctx.config;
     }
 
+    protected async checkYaml(ctx: Context): Promise<void> {
+        // always add the YAML extension to support .gitpod.yml editing
+        this.addExtension(ctx, 'redhat.vscode-yaml');
```

Comment:
We already always install it in VS Code.

Reply:
Removed it 🙏